src/otx/algo/callbacks/adaptive_train_scheduling.py

```diff
@@ -62,6 +62,16 @@ def on_train_start(self, trainer: Trainer, pl_module: LightningModule) -> None:
             trainer.check_val_every_n_epoch = adaptive_check_val_every_n_epoch
             self._change_early_stopping_patience(trainer.callbacks, adaptive_check_val_every_n_epoch)
             self._change_lr_scheduler_frequency(trainer.lr_scheduler_configs, adaptive_check_val_every_n_epoch)
+        else:
```

Comment:
Why is this under `else` branch?

Reply:
When if condition is satisfied, it already considers the OTX1.x case.
https://github.com/openvinotoolkit/training_extensions/blob/e0d627d8f5eec36aee793e0a426c50a5bd7c5998/src/otx/algo/callbacks/adaptive_train_scheduling.py#L137

Reply:
Fixed the logic as we discussed, https://github.com/openvinotoolkit/training_extensions/pull/3060/commits/827720616db9ddb9b367f574254464abe34eb468